core/src/main/scala/scalan/arrays/ArrayOps.scala

```diff
@@ -297,12 +301,11 @@ trait ArrayOpsStd extends ArrayOps {
 
   def array_rangeFrom0(n: Rep[Int]): Arr[Int] = 0.until(n).toArray
 
-  def array_filter[T](xs: Array[T], f: T => Boolean): Array[T] =
-    genericArrayOps(xs).filter(f)
+  def array_filter[T](xs: Array[T], f: T => Boolean): Array[T] = genericArrayOps(xs).filter(f)
```

Comment:
I just wanted to make it one-line, if you ask about the change. If not, please clarify what do you mean.